presto-main/src/main/java/com/facebook/presto/sql/analyzer/ExpressionAnalyzer.java

```diff
@@ -1093,6 +1096,18 @@ protected Type visitNode(Node node, StackableAstVisitorContext<Context> context)
             throw new SemanticException(NOT_SUPPORTED, node, "not yet implemented: " + node.getClass().getName());
         }
 
+        public Type visitGroupingOperation(GroupingOperation node, StackableAstVisitorContext<Context> context)
+        {
+            if (node.getGroupingColumns().size() > MAX_NUMBER_GROUPING_ARGUMENTS) {
+                throw new SemanticException(INVALID_PROCEDURE_ARGUMENTS, node, "GROUPING supports up to 63 column arguments");
```

Comment:
Use the constant to construct the message so that it doesn't get out of sync if we adjust its value.